projects/elife.yaml

```diff
@@ -918,6 +918,32 @@ reviewer-suggestions:
             1241: 80
         ram: 4096
 
+peerscout:
+    description: PeerScout
```

Comment:
not much of a description like for many other projects :smile_cat:

Reply:
Want to keep it consistent ;-)